inst/include/RcppArmadilloAs.h

```diff
@@ -77,7 +77,9 @@ namespace traits {
     private:
         MATRIX mat ;
     };
-         
+    
+    // 14 June 2017
```

Comment:
I just followed the style of the source code. There are also dates [there](https://github.com/RcppCore/RcppArmadillo/blob/master/inst/include/RcppArmadilloAs.h#L116)